### ISSUE TITLE
OGG files play very laggy

### ISSUE THREAD

**Issue by Chaosphere** [kabouzeid/phonograph-issue-tracker#3]
When i listen to a ogg file the sound is very laggy it stutters.

**Comment by kabouzeid** [comment]
Does that happen with other players? And which other players did you try?

**Comment by Chaosphere** [comment]
No, other players work well. I've tryed with Play Music, Tomahawk, Encore and VLC, all of them work just fine with ogg files.

**Comment by kabouzeid** [comment]
Would you mind trying shuttle?

**Comment by Chaosphere** [comment]
Shuttle works well. I've just tryed it as you requested.

**Comment by kabouzeid** [comment]
Alright, I don't know whats the problem then :/ I will let you know if I find something related

**Comment by Chaosphere** [comment]
Ok, thank you :)
